src/test/scala/htsjdk/UnitSpec.scala

```diff
@@ -0,0 +1,6 @@
+package htsjdk
+
+import org.scalatest.{FlatSpec, Matchers}
+
+/** Base class for all Scala tests. */
+class UnitSpec extends FlatSpec with Matchers
```

Comment:
The idea is that scalatest offers different styles of test writing, and one might choose to use one style for unit testing, and a different one for integration or larger functional testing.  So the convention is to name the base class by the type of test it is intended for, in this case unit testing.